remoteappmanager/static/js/home/views/application_list_view.js

```diff
@@ -51,14 +45,29 @@ define([
               '              :disabled="app.is_stopping()">' +
               '        <i class="fa fa-times"></i>' +
               '      </button>' +
-
               '      <span>{{ app.app_data.image | app_name }}</span>' +
               '    </a>' +
               '  </li>' +
               '</ul>' +
               '<!-- /.sidebar-menu -->' +
             '</section>' +
-            '<!-- /.sidebar -->'
+            '<!-- /.sidebar -->',
+
+        data: function() {
+            return { 'search_input': '' };
+        },
+
+        computed: {
+            'entries_visible': function() {
```

Comment:
you don't need a visibility array. What you need is a filtering operation over the model vs represented model.